packages/dependency-extraction-webpack-plugin/src/index.js

```diff
@@ -22,6 +22,7 @@ const wooRequestToExternal = ( request ) => {
 		const handle = request.substring( WOOCOMMERCE_NAMESPACE.length );
 		const irregularExternalMap = {
 			'blocks-registry': [ 'wc', 'wcBlocksRegistry' ],
+			'blocks-settings': [ 'wc', 'wcSettings' ],
```

Comment:
blocks settings is just `wcSettings` without the wc prefix.
![image](https://user-images.githubusercontent.com/6165348/106666000-4cf4fd80-65a7-11eb-8504-b5c9ba62bec8.png)

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/6228/commits/fec29a4c4bcc189137527d4b34ecfc07c28fc284